index.html

```diff
@@ -6,92 +6,103 @@
         <title>Delicious</title>
         <meta name="description" content="Dessert Recipe">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link rel="stylesheet" href="css/normalize.css">
-        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="css/layout.css">
+        <link rel="stylesheet" href="css/base.css">
```

Comment:
Once you've removed the Skeleton base styles from the layout.css, make sure to load base.css first.